models/gpt2/gpt2.py

```diff
@@ -275,6 +275,10 @@ def get_activation_fn(params):
         return mtf.selu
     elif activation_fn == "elu": # https://arxiv.org/abs/1511.07289
         return mtf.elu
+    elif activation_fn == "lrelu(0.01)":
```

Comment:
names should only contain lowercase letters and underscores. Do you think something like lrelu001 would be a good name?

Reply:
lrelu001 is not readily understood though, might want to also add a comment regarding the same.